packages/fiori/src/UploadCollection.ts

```diff
@@ -41,10 +41,6 @@ type SelectionChangeEventDetail = {
 	selectedItems: Array<UploadCollectionItem>,
 };
 
-type ItemDeleteEventDetail = {
-	item: UploadCollectionItem,
-};
```

Comment:
What is the intent for removing the type annotation from the event?

Reply:
It was used by the UploadCollectionItem to send private information to the UploadCollection. This code is now deleted. See @ui5-item-delete and @ui5-_uci-delete in UploadCollection.hbs

Reply:
As we discussed this was not a useful type, because the public event was not using it. Now I changed that.